src/app/(dashboard)/teams/[teamId]/page.tsx

```diff
@@ -189,20 +189,7 @@ export default function Page({ params }: PageProps) {
                 </div>
 
                 <div className="pt-4 grid gap-4 grid-cols-1 md:grid-cols-2 lg:grid-cols-3">
-                  <Card className="col-span-1 md:col-span-1">
-                    <CardHeader>
-                      <CardTitle>Github Repositories</CardTitle>
-                      <CardDescription>
-                        You have {registeredGitRepos.length} github repositories
-                        registered
-                      </CardDescription>
-                    </CardHeader>
-                    <CardContent>
-                      <GithubRepoList githubRepos={registeredGitRepos} />
-                    </CardContent>
-                  </Card>
-
-                  <div className="col-span-2 md:col-span-2">
+                  <div className="col-span-3 md:col-span-3">
```

Comment:
We are removing the repo list from the team page completely?

Reply:
Yeah, I'm leaving it only with the charts, I will put a better screenshot that also shows that on the PR.
Happy to be challenged if this was a good decision, but IMHO looks better